recipes/deodr/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "deodr" %}
+{% set version = "0.1.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/martinResearch/DEODR/releases/download/v0.1.14/deodr-0.1.14.tar.gz
+  sha256: 8e559df8e119d949994213a1f1ae77210def7ccfa248dc8ceeabd9505e389772
+  patches:
+    # Currently the required CXX file for build isn't packaged in the release, so generated here.
+    - setup_CXX.patch
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} setup.py build_ext --inplace
```

Comment:
Do we need to do the `build_ext` separately? I would think the `pip install` on the next line would be sufficient to build the extension modules.